### ISSUE TITLE
Supervisor Meeting - 05/11/24

### ISSUE THREAD

**Issue by jsontran** [SumanyaG/Alkalytics#130]
**Date and Time**
05/11/24 - 3:30

**Attendance**
- [ ] Bassel Abdulkader
- [ ] Sumanya Gulati
- [ ] Kate Min
- [ ] Jennifer Ye
- [ ] Jason Tran

**Agenda**
- VnV Update; send over documents
- Update on POC
- What to expect this week

**Questions to Ask**
N/A

**Comment by SumanyaG** [comment]
<img width="485" alt="Screenshot 2024-11-05 at 4 25 06 PM" src="https://github.com/user-attachments/assets/5edf9e02-7a78-4620-a6ee-50af5bec5b7e">

the plan so far!

**Comment by katemin991** [comment]
**meeting minutes nov 5 2024 with supervisor**
- our updates: finished vnvplan document, working on demo
    - TODO: email the vnvplan doc

project concerns n clarifications:
1. database hosting: can host on their lab(?) pc; storage should not be an issue
- bassel's concerns on transferring the db to a different computer later on: tis possible but a lil difficult, may want to consider transferring to cloud (would be more costly tho)
2. data migration: how the data is meant to be organized
- one "master" spreadsheet has conditions and parameters of each experiment performed on x date
  - one spreadsheet = one specific experiment that has date + timestamp + detailed parameters/values
  - one row on the "master" corresponds to one experiment spreadsheet n some experiment dates in the master are the same date - need to be able to uniquely identify them tho
  - any sheets w/ rows of 0s or missing rows should be flagged and cleaned up
  - we cannot allow user to manipulate db directly tho
  - emphasis on the data analysis n machine learning stuff once migration is done
    - might need to update the stretch goals? (i forget what our stretch goals are so this comment could potentially be ignored) 

- bassel will share several experiment sheets for migration
- our demo to the supervisors on the 12th is still part of upcoming plans

**Comment by katemin991** [comment]
**meeting minutes -- internal team meeting nov 5 2024**
- updated plan for poc demo:
  - jason do algo
  - kate set up db
  - jenn test that it works
  - sumanya handles most of poc contribution report

- poc contribution report stuff:
  - demo plan + team/supervisor meeting attendance sections : kate
  - lecture + ta informal pres attendance sections : jenn
  - commit logs + issue tracker + cicd sections : sumanya
    - **commit log**: count by how many PRs each person made to the milestone branches instead; explain why we not using the tracking method they ask for in template
      - n only count authors
    - **issue tracker**: just list who did what sections; explain why we not doing it the template way (meeting chairs made dev/vnv plan issues, srs was individual, etcetc so TLDR: not an effective way to show actual contributions) 
      - write the tracking method we opted to do as a separate section
    - **cicd**: mention the latex one we have rn, maybe mention future additions?

- other stuff:
  - complete team contribution survey (individual) on avenue by nov 6
  - do VnVPlan peer reviews by nov 7/8 probably
